backend/tests/test_app_check_config.py

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
I'd fold this file into `test_cpr.py` — they seem quite related, and both are rather small anyway…

Reply:
I agree!, DUMMY_MODE is a bad name, but general: It relays the information that all integrations are turned off, which is what I think is the meaning: The inverse of PROD_MODE (where all integrations are ago) - we just have only one at present.

Could We agree on another name: DEMO_MODE / STUB_MODE

I will put in checks for the 4 uuids as well and move the tests into test_cpr.